docs/cookbook/using-docker.md

```diff
@@ -0,0 +1,299 @@
+Next, the `.env` file should be modified to contain values for each of the new environment variables. Each will get populated with values specific to your S3 buckets. Again, add the `APOS_S3_ENDPOINT` with value if using a service not hosted by AWS.
+
+<AposCodeBlock>
+
+```sh
+NODE_ENV=production
+APOS_MONGODB_URI=mongodb://db:27017/apostrophe
+APOS_S3_REGION=<your region>
+APOS_S3_BUCKET=<your bucket name>
+APOS_S3_KEY=<account key>
+APOS_S3_SECRET=<account secret>
+```
+
+<template v-slot:caption>
+  .env
+</template>
+
+</AposCodeBlock>
+
+In addition to these changes, you will also likely have to change your `@apostrophecms/uploadfs` options for accessing the bucket. For example:
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  options: {
+    uploadfs: {
+      bucketObjectsACL: 'bucket-owner-full-control'
+    }
+  }
+};
+```
+
+<template v-slot:caption>
+  modules/@apostrophecms/uploadfs/index.js
+</template>
+
+</AposCodeBlock>
+
+### Finishing up
+Just like with the Docker container previously, you can now bring the site up with:
+
+```cli
+docker composer up
+```
+Any assets uploaded through the site will now be stored in your S3 bucket rather than on the server directly.
+
+## Using MongoDB Atlas
+[MongoDB Atlas](https://www.mongodb.com/atlas/database) is a robust, multi-cloud database service. There are a number of advantages, but one is that using a cloud database means that our project can run on multiple servers but still all access the same database. If we run our project in a docker container without an accompanying database container, we don't have to use Docker Compose. However, since we have already built these assets, we can continue with these files. Two files must be altered - `docker-compose.yaml` and `.env`.
+
+### Changing the `docker-compose.yaml` file
+Since we no longer have to use the database container, we can simply delete that whole section from the `services:`.
+
+### Changing the `.env` file
+First, start by getting an account and setting up a project and cluster according to the [instructions](https://www.mongodb.com/docs/atlas/?_ga=2.115258319.959071482.1662986164-305956368.1655805952&_gac=1.50376795.1662898658.Cj0KCQjwjvaYBhDlARIsAO8PkE2KG3UP3yszcTYrzDpB8BRxDZ7vM2vLMafvX59emZZKkDExo_ZPZRIaAneGEALw_wcB) at the Atlas site. Once you do this, you can get the connect string for your database. The `APOS_MONGODB_URI` was already being set within the `.env` file. You simply need to substitute your connect string for the value.
+
+::: note
+Any special characters in your user name or password within the connection string need to be converted to %-encoding.
+:::
+
+### Finishing up
+Since this will create a new database, once you bring your site up you should add an admin user as was [detailed](#spinningourprojectup) when we used the containerized version.
+
+To bring the site up use :
+```cli
+docker composer up
+```
+
+To bring it down:
+```cli
+docker composer down
+```
+
+## Deploy
```

Comment:
We could just say that you can run these commands on any server that supports Docker, and that you can put a load balancer in front of several such servers if you use MongoDB Atlas and S3, in which case you should host your containers within AWS in the same region with your Atlas cluster and S3 bucket. In that case, using Lightsail container hosting may be the simplest option.

Reply:
We need an "updating your project" section, covering how to update the container image and restart when your code or dependencies have changed.

Reply:
Added, but might need some work.